src/patternfly/components/ProgressStepper/examples/ProgressStepper.md

```diff
@@ -244,11 +244,11 @@ cssPrefix: pf-v6-c-progress-stepper
       {{/progress-stepper-step-description}}
     {{/progress-stepper-step-main}}
   {{/progress-stepper-step}}
-  {{#> progress-stepper-step progress-stepper-step--IsInProgress="true" progress-stepper-step--IsCurrent="true"}}
+  {{#> progress-stepper-step progress-stepper-step--IsInProgressAnimate="true" progress-stepper-step--IsCurrent="true"}}
     {{> progress-stepper-step-icon}}
     {{#> progress-stepper-step-main}}
       {{#> progress-stepper-step-title}}
-        Second step
+        In process
```

Comment:
WDYT about changing this to "In progress" - both here and in this other spot https://github.com/patternfly/patternfly/blob/e25a7756fa79ab47799c2a3ee4d44a31feb36919/src/patternfly/components/ProgressStepper/examples/ProgressStepper.md?plain=1#L537